keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ApplicationEvent.kt

```diff
@@ -16,9 +16,10 @@ import java.time.Instant
   JsonSubTypes.Type(value = ApplicationActuationResumed::class, name = "ApplicationActuationResumed")
 )
 abstract class ApplicationEvent(
-  override val triggeredBy: String? = null
+  override val triggeredBy: String? = null,
+  override val message: String? = null
```

Comment:
Added `message` to the `PersistentEvent` contract, which, along with `triggeredBy`, create a nice easy contract for the UI around basic properties every event we'll return should always have. The defaults are `null` in abstract classes, but sub-classes are encouraged to implement a meaningful message.